students/ABirgenheier/lesson03/slicing.py

```diff
@@ -0,0 +1,67 @@
+def first_last_exchange(n):
+    if len(n) > 1:
+        result = n[-1] + n[1:-1] + n[0]
```

Comment:
`n[0]` or `n[-1]` is using indexing - please update code to use slice index only